src/services/ldap-config/service.js

```diff
@@ -0,0 +1,288 @@
+const mongoose = require('mongoose');
+const reqlib = require('app-root-path').require;
+
+const errorHandlers = require('./errors');
+
+const { Forbidden } = reqlib('src/errors');
+
+class LdapConfigService {
+	setup(app) {
+		this.app = app;
+		this.systemId = undefined;
+	}
+
+	/**
+	 * `GET /ldap-config/:systemId`
+	 * Returns the ldapConfig of a given system in the same format as accepted
+	 * by the other methods of this service.
+	 * If the given system is not a LDAP system (i.e. does not have the ldapConfig
+	 * attribute) or is not of the type `general` (i.e. a self-service config), the
+	 * access is denied.
+	 * To access the system data, the authentication information are proxied to the
+	 * System service, so that access is only granted to administrators and only to
+	 * systems of the own school.
+	 * @param {ObjectId} systemId ID of an existing system with LDAP config
+	 * @param {Object} params Feathers request params
```

Comment:
I would put the reason of the params usage to the inline docs instead of using comments in the function.
Moreover I think there are too much comments for this function.

Reply:
It is pretty clear what the function is doing. The comments should describe what is not visible from the code itself.

Reply:
While I agree it is clear what the function is doing, I think there is value in good documentation here.  Itmight be valuable to move this to the openapi docs in the future. For an old-style service, I think this is okay for now, but I am happy to comply; what do you think @Metauriel ?